packages/0x.js/package.json

```diff
@@ -92,14 +87,12 @@
     "bignumber.js": "~4.1.0",
     "bn.js": "4.11.8",
     "compare-versions": "^3.0.1",
-    "es6-promisify": "^5.0.0",
```

Comment:
don't we still rely on `es6-promisify` in this package?

Reply:
We do. Good catch.